internal/common/lexer.go

```diff
@@ -132,19 +166,57 @@ func (l *Lexer) Location() errors.Location {
 	}
 }
 
+func (l *Lexer) consumeTripleQuoteComment() {
+	if l.next != '"' {
+		panic("consumeTripleQuoteComment used in wrong context: no third quote?")
+	}
+
+	if l.descComment != "" {
+		l.descComment += "\n"
+	}
+
+	comment := ""
+	numQuotes := 0
+	for {
+		l.next = l.sc.Next()
+		if l.next == '"' {
+			numQuotes++
+		} else {
+			numQuotes = 0
+		}
+		comment += string(l.next)
+		if numQuotes == 3 || l.next == scanner.EOF {
+			break
+		}
+	}
+	l.descComment += strings.TrimSpace(comment[:len(comment)-numQuotes])
+}
+
+func (l *Lexer) consumeStringComment(str string) {
+	if l.descComment != "" {
+		l.descComment += "\n"
```

Comment:
Not covered by unit tests. Can you add a test case for this, please?

Reply:
Actually this case doesn't make sense. A one line string comment will always be the only description.